frontend/lib/identification/animated_progressbar.dart

```diff
@@ -21,7 +21,9 @@ class _AnimatedProgressbarState extends State<AnimatedProgressbar>
   void initState() {
     super.initState();
     controller = AnimationController(
-        duration: Duration(milliseconds: widget.duration), vsync: this);
+        duration: Duration(
+            milliseconds: widget.duration > 0 ? widget.duration : 1 << 63),
```

Comment:
What does this bitshift? Maybe we could have a named constant for that. Or can we just leave away the animation if the duration is leq 0?

Reply:
I will leave away the animation. There is no max integer constant in dart, thats why I used the bit shift.